code/modules/projectiles/guns/ballistic/bow.dm

```diff
@@ -269,9 +269,53 @@
 	zoomable = TRUE
 	zoom_amt = 10
 	zoom_out_amt = 5
-	pin = /obj/item/firing_pin/implant/pindicate
+	pin = /obj/item/firing_pin
 	fire_sound = null
 	draw_sound = null
+	var/folded = FALSE
+	var/stored_ammo ///what was stored in the magazine before being folded?
+
+/obj/item/gun/ballistic/bow/energy/syndicate/shoot_live_shot(mob/living/user, pointblank, atom/pbtarget, message)
+	if(folded == FALSE)
+		. = ..()
+	else
+		to_chat(user, span_notice("You must unfold [src] before firing it!"))
+		return FALSE
+
+/obj/item/gun/ballistic/bow/energy/syndicate/attack_self(mob/living/user)
+	if(folded == FALSE)
+		. = ..()
+	else
+		to_chat(user, span_notice("You must unfold [src] to chamber a round!"))
+		return FALSE
+
+/obj/item/gun/ballistic/bow/energy/syndicate/AltClick(mob/living/user)
+	if(folded == FALSE)
+		. = ..()
+	else
+		to_chat(user, span_notice("You must unfold [src] to switch firing modes!"))
+		return FALSE
+
+/obj/item/gun/ballistic/bow/energy/syndicate/CtrlClick(mob/living/user)
+	if(!user.is_holding(src))
+		to_chat(user, span_notice("You need be holding [src] to do that!"))
+		return
+	folded = !folded
+	playsound(src.loc, 'sound/weapons/batonextend.ogg', 50, 1)
+	if(folded)
+		to_chat(user, span_notice("You fold [src]."))
+		w_class = WEIGHT_CLASS_NORMAL
+		chambered = null
+		icon_state = "bow_syndicate_folded"
+		stored_ammo = magazine.ammo_list()
+		magazine.stored_ammo = null
+		update_icon()
+	else
+		w_class = WEIGHT_CLASS_BULKY
+		to_chat(user, span_notice("You extend [src], allowing it to be fired."))
+		icon_state = "bow_syndicate"
+		magazine.stored_ammo = stored_ammo
+		update_icon()
```

Comment:
```suggestion
	if(folded)
		to_chat(user, span_notice("You fold [src]."))
		w_class = WEIGHT_CLASS_NORMAL
		chambered = null
		stored_ammo = magazine.ammo_list()
		magazine.stored_ammo = null
	else
		w_class = WEIGHT_CLASS_BULKY
		to_chat(user, span_notice("You extend [src], allowing it to be fired."))
		magazine.stored_ammo = stored_ammo
	update_icon()

/obj/item/gun/ballistic/bow/energy/syndicate/update_icon()
	. = ..()
	folded ? icon_state = "bow_syndicate_folded" : "bow_syndicate"
```

Reply:
suggestion does not compile

Reply:
i inverted the icon state with the folded it should work now